README.md

```diff
@@ -80,8 +80,7 @@ Custom attr:
  * `pstsPaddingMiddle` If true, the tabs start at the middle of the view (Like Newsstand google app)
  * `pstsTextStyle` Set the text style, default bold
  * `pstsTextSelectedStyle` Set the text style of the selected tab, default bold
- * `pstsTextAlpha` Set the text alpha transparency, default 0.5
- * `pstsTextSelectedAlpha` Set the text alpha transparency of the selected tab, default 1
+ * `pstsTextColorInactive` Set text color of inactive text so you can use ARGB without using `View.setAlpha`
```

Comment:
Because it's a custom view, you're certainly free to use only custom attributes and ignore the existing ones.

For me, it makes sense to use the existing ones, and append where necessary.

> but then you always have to define a transparency colour in `android:textColor` which is kind of weird

Why do you need to define a transparency color? You can still use a plain RGB in `android:textColor` and a plain RGB in `pstsTextColorSelected` which is used when the tab is selected - I'm not sure I see why alpha is required, maybe I misunderstand you.

**Ideally, `android:textColor` would be enough - there wouldn't need to be another attribute for selected text color because a color state list is a valid input for this attribute so developers should be able to provide a "selected" state.**